### PR TITLE
[docs] Improve RaisinBread Docs

### LINKED ISSUE #6935: [docs] improve Raisinbread install docs
Based on Haowei's observations when setting up his sandbox this past week --
A few clarifications would be useful in the Raisinbread setup instructions in the repo markdowns. 
The internal-only VM setup googledoc contains notes that perhaps the raisinbread readme could use. 

Welcome to your first issue for Google Season of Docs, @haoweiqiu ! 
Important to keep in mind: the repo markdowns are public-facing, and so should not contain any information specific to MCIN / Loris-team internal development setup.  

cc: @laemtl @AlexandraLivadas  whose input will be useful on this, and who both worked on related docs recently. 

(I can't officially make Haowei's handle the "assignee" until samir adds him on github but that will come very soon)

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [X] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #6935  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: raisinbread/README.md
Comment (christinerogers) [inline raisinbread/README.md:65]: wondering - why add `source` if it's not mentioned in the example below?

File: raisinbread/README.md
Comment (christinerogers) [inline raisinbread/README.md:68]: Hi Haowei, the table order for loading was specified because it matters.   
If the loading goes wrong, it's hard to debug without knowing exactly what the table order was. Instead of removing this detail, let's instead add a comment mentioning _why_ the exact table order is specified --  
Something like "It's important to load the Raisinbread tables in the order listed below - here is a command for you to use"  (i'm writing this quickly, I'm sure you'll make the actual edit sound better)

File: raisinbread/README.md
Comment (christinerogers) [inline raisinbread/README.md:47]: mmmm i think those details were previously omitted/underspecified because the _admin_username_ and possibly localhost argument mean the user can't just copy-paste this line. If you think this genuinely makes it clear for users and you've ensured it's consistent with how our other documentation ( docs/ ) invokes the mysql commandline, go for it.

Comment (ridz1208) [reply raisinbread/README.md:47]: It was indeed omitted because the assumptions section above explains the command structure https://github.com/aces/Loris/pull/6937/files#diff-891d4b84b8f410eef353c238c8045ab6L35

File: raisinbread/README.md
Comment (ridz1208) [reply raisinbread/README.md:65]: this is an unnecessary change

**Review by ridz1208 — DISMISSED** [review]

File: raisinbread/README.md
Comment (haoweiqiu) [reply raisinbread/README.md]: I added a note here to mention the importance of tables order based on your advice.

File: raisinbread/README.md
Comment (haoweiqiu) [reply raisinbread/README.md]: @ridz1208 Thank you for the comment! I have removed the changes.

File: raisinbread/README.md
Comment (laemtl) [inline raisinbread/README.md:53]: tables by manually (extra space)

File: raisinbread/README.md
Comment (laemtl) [inline raisinbread/README.md:56]: I suggest to remove the path /var/www/loris/ (next line also). It's setup dependent and as the next commands below are omitting it, we suppose the user cd there first.

Comment (haoweiqiu) [reply raisinbread/README.md]: Thanks for the advice.

File: raisinbread/README.md
Comment (AlexandraLivadas) [inline raisinbread/README.md:53]: I would specify here that these tables need to be dropped in this order. If you try to source "9999-99-99-drop_tables.sql" first, there will be errors.

**Review by AlexandraLivadas — DISMISSED** [review]

File: raisinbread/README.md
Comment (haoweiqiu) [reply raisinbread/README.md:53]: Thanks for the comment. I have added a note here.

**Comment by christinerogers** [comment]
waiting on re-review from @ridz1208  - comments should be addressed, Rida. 
(cc @AlexandraLivadas if you have time to check your comments were addressed)

(@laemtl i'm dismissing your approval and requesting re-review because of 2 pushes since)

**Review by laemtl — APPROVED** [review]

**Review by christinerogers — APPROVED** [review]
Approving.  may have minor edits later.  

@ridz1208 could you do a final review re Raisinbread sourcing?

**Comment by christinerogers** [comment]
@ridz1208 @driusan  final review on these RaisinBread docs? thank you.

**Comment by cmadjar** [comment]
@haoweiqiu can you please rebase your PR so that the checks can run.

@driusan could you review once rebase?

**Comment by ridz1208** [comment]
@driusan rebased, ready for you

**Pull request merged**
